src/main/java/com/shima/patientchart/service/PatientChartService.java

```diff
@@ -15,9 +18,36 @@ public PatientChartService(PatientChartMapper patientchartMapper) {
         this.patientchartMapper = patientchartMapper;
```

Comment:
ここもキャメルケースの命名の統一感を守りましょう！
```suggestion
        this.patientChartMapper = patientChartMapper;
```